docs/Developers/Git_Basics.rst

```diff
@@ -186,7 +186,7 @@ You are awesome!
     to sync your local fork with the remote repository before sending
     a pull request.
 
-    More information regarding syncing can be found `here <http://coala.readthedocs.io/en/latest/Developers/Git_Basics.html#rebasing>`_.
+    More information regarding syncing can be found `here <http://docs.coala.io/en/latest/Developers/Git_Basics.html#rebasing>`_.
```

Comment:
I think instead of 
`<http://docs.coala.io/en/latest/Developers/Git_Basics.html#rebasing>`
it can be
`<Developers/Git_Bascis#rebasing>`

Please try that